README.md

```diff
@@ -70,11 +130,8 @@ gaiad init
 You can find the corresponding genesis files [here](https://github.com/tendermint/testnets). Then replace the `genesis.json`and `config.toml` files:
```

Comment:
This deserves an explanation. 
- Explain what the genesis files are for, and how to choose the correct ones. "Genesis files are the starting point for the full-node to sync up with the network. In order to sync up with the correct version of the Testnet, be sure to choose the genesis file corresponding to the version of the Testnet you want to connect to."
- Short explanation of what the config.toml file is for.